File: twiml/voice/gather/gather-4/gather-4.5.x.cs
using System;
using Twilio.TwiML;
using Twilio.TwiML.Voice;
using System.Linq;

class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        var gather = new Gather(input: new []{Gather.InputEnum.Speech}.ToList(),
            action: new Uri("/completed"));
        gather.Say("Welcome to Twilio, please tell us why you're calling");
        response.Append(gather);

        Console.WriteLine(response.ToString());
    }
}

File: twiml/voice/gather/gather-4/gather-4.5.x.rb
require 'twilio-ruby'

response = Twilio::TwiML::VoiceResponse.new 
gather = Twilio::TwiML::Gather.new input: 'speech', action: '/completed'
gather.say 'Welcome to Twilio, please tell us why you\'re calling'
response.append gather

puts response
